mailpoet/assets/js/src/automation/integrations/mailpoet/steps/send_email/index.tsx

```diff
@@ -7,6 +7,8 @@ export const step: StepType = {
   group: 'actions',
   title: 'Send email',
   description: 'An email will be sent to subscriber',
-  icon: Icon,
+  subtitle: (data) => `Email ID ${data.args.email_id as string}`,
+  color: '#996800',
+  icon: Icon(),
```

Comment:
I'm wondering why we now need to call the component function here. I'm a bit surprised it works as we're not in a React tree here yet. That's why I think we should pass component functions (like before) or render callbacks (like for the subtitle). I see this works now but still looks slightly suspicious to me.

Reply:
That was an odd one. I switched to @wordpress/components for the icon and was able to get away from `JSX.Element` and back to `ComponentType` ([a3893c4](https://github.com/mailpoet/mailpoet/pull/4275/commits/a3893c46d247eaf9842cdd1b27c63a419e9b6722))